docs-src/app-dev-context/add-sdk.md

```diff
@@ -1,8 +1,8 @@
 ---
 id: add-sdk
-title: How to add a Temporal SDK to your project
+title: How to install a Temporal SDK to your project
```

Comment:
Now that we're using "install," I think we can drop "to your project" from the title.
```suggestion
title: How to install a Temporal SDK
```